jbehave-core/src/main/java/org/jbehave/core/steps/ParameterConverters.java

```diff
@@ -307,7 +309,8 @@ private ParameterConverters(StepMonitor monitor, List<ParameterConverter> conver
                 new FunctionalParameterConverter<>(String.class, YearMonth.class, YearMonth::parse),
                 new FunctionalParameterConverter<>(String.class, ZonedDateTime.class, ZonedDateTime::parse),
                 new FunctionalParameterConverter<>(String.class, ZoneId.class, ZoneId::of),
-                new FunctionalParameterConverter<>(String.class, ZoneOffset.class, ZoneOffset::of)
+                new FunctionalParameterConverter<>(String.class, ZoneOffset.class, ZoneOffset::of),
+                new FunctionalParameterConverter<>(String.class, Path.class, Paths::get)
```

Comment:
please see comment above:
>// java.time.* converters

so either add an empty line between set of time-converters and new one, or move new converter higher in the collection